### ISSUE TITLE
Feature suggestion: hide action buttons until hover (with accompanying CSS)

### ISSUE THREAD

**Issue by appel** [kevinwucodes/gmail-quick-links#49]
Hey Kevin,

Thanks so much for a great and indispensable extension!

I modified the way the action buttons on the right (edit, rename and delete) are displayed using the Stylus extension and figure I'd share it just in case you happen to like the change. All I did was hide the 3 icons by default by setting the opacity to 0, then on hover of the parent div set the opacity of the 3 icons to .7, and on hover of an icon set it to 1.

I thought this helps make it feel a little less cluttered, plus it mimics the way Gmail itself handles some inline action buttons when you hover over a message in your inbox (like Archive, Delete, etc.):

![image](https://user-images.githubusercontent.com/61596/88100182-3090f680-cb6a-11ea-8364-917862035b87.png)

Here's a screenshot, hopefully clarifying the behavior:

![image](https://user-images.githubusercontent.com/61596/88099556-54077180-cb69-11ea-8787-946b926f188d.png)

As you can see, there are two Quick Links. My mouse is hovering over both the second link and the middle icon, so all icons for this link are shown (with opacity 0.7) and the middle one has an opacity of 1.

Here's the CSS:

    #listContainer div div span.glyph {
		opacity: 0;
    }

    #listContainer div div:hover span.glyph {
		opacity: 0.7;
    }

    #listContainer div div:hover span.glyph:hover {
		opacity: 1;
    }

Hope this is useful! Thanks again for open sourcing your extension!

**Comment by Zanatoo** [comment]
Hi Appel, 

You can achieve this yourself quite easily with a Chrome extension called [Sytlebot].(https://chrome.google.com/webstore/detail/stylebot/oiaejidbmkiecgbjeifoejpgmdaleoha)

It allows you to add/overwrite any CSS to pages.

**Comment by appel** [comment]
Thanks for the suggestion, @Zanatoo. As I wrote in my post I'm actually already using a chrome extension called [Stylus](https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne) to do this. I just wanted to share my modifications here for others.

**Comment by Zanatoo** [comment]
Doh! My apologies. I should have read your post properly.

**Comment by appel** [comment]
No worries, I appreciate you trying to help.

**Comment by joxerg** [comment]
So much clean result!! would be nice to have this piece of software more integrated in Gmail's style: replace the "Add quick link" qith a "+" icon (like the Creta enew label one), change the blue "i" icong color, modify margins of links...
Thank you!!

**Comment by joxerg** [comment]
Played with some CSS, so much cleaner :) (still can't remove "underline" or visited link purple color...)
![CAPT- 3 16 28 PM](https://user-images.githubusercontent.com/28994304/120071735-e84b3a80-c090-11eb-84cb-39655fec5c75.png)

**Comment by appel** [comment]
@joxerg Try this for the color and underline:

```
div#gmailQuickLinks a {
	color: inherit;
	text-decoration: none !important;
}
```

**Comment by joxerg** [comment]
Wow thank you very much appel!! am trying to have active link bold (like Gmail's when clicked), no luck...any ideas?

Tried (only bold the instant i click, then goes normal):
div#gmailQuickLinks a:active {
  font-weight: bold !important;
}

**Comment by appel** [comment]
There's no (reliable) way to apply styles for an active state because the anchors have no active classes. That's something that would need to be added to the extension.

The closest I could get is to use the :active pseudo selector via the code below, but it's only visible onclick.

```
div#gmailQuickLinks a:active {
	font-weight: bold;
}
```

Edit: oops, misread your code, thought you tried using an .active class, apologies.

**Comment by joxerg** [comment]
Yep thank you anyway!! well, looked into de default.css of the extension but did not find any "div#gmailQuickLinks", if found we could load the extension out of Chrome store...Anyway, so much clean-nice now:

![CAPT- 11 10 33 PM](https://user-images.githubusercontent.com/28994304/120084753-2701e480-c0d3-11eb-9475-b7548e611778.png)
